app/models/levelling_up_premium_payments/eligibility.rb

```diff
@@ -84,7 +83,7 @@ def eligible_later?
     end
 
     def award_amount
-      calculate_award_amount
+      super || calculate_award_amount
```

Comment:
I had a weird bug where if it called `super` first then showed the award amount, if I revisited the page, the award amount would increase. I didn't really understand it, but getting rid of the call to `super` fixed it. You'll want to try coming back to the page and making sure the monetary amounts don't change.

Reply:
Please try it locally! I only saw this when I hit the back button and came back to the page and the award amount doubled. I never really understood why that was happening.

Reply:
I've been back and forward through this page hundreds of times during the making of the PR, trust me ;)

Reply:
Ha! Fair enough!